API_REFERENCE.md

```diff
@@ -166,6 +166,12 @@ iamRole
 - description: IAM ARN to be assumed by the container for permissions to cloud resources
 - requirement: must be a valid ARN
 
+dns
+- type:        String
+- required:    false
+- description: What type of DNS server to use foamr a Shipment.
```

Comment:
to use "from"

Reply:
`What type of DNS server to use for a Shipment`